packages/web-components/src/label/label.template.ts

```diff
@@ -0,0 +1,15 @@
+import { ElementViewTemplate, html } from '@microsoft/fast-element';
+import { Label } from './label.js';
+
+/**
+ * The template for the Fluent label web-component.
+ * @public
+ */
+export function labelTemplate<T extends Label>(): ElementViewTemplate<T> {
+  return html<T>`
+    <slot></slot>
+    <span part="asterisk" class="asterisk" ?hidden="${x => !x.required}">*</span>
```

Comment:
Just for me to learn - is it better to use `hidden` attribute than conditionally rendering the span?